garrysmod/gamemodes/sandbox/entities/weapons/gmod_tool/stools/balloon.lua

```diff
@@ -84,8 +86,10 @@ function TOOL:LeftClick( trace, attach )
 	local Offset = CurPos - NearestPoint
 
 	local Pos = trace.HitPos + Offset
+	local Ang = trace.HitNormal:Angle() + hitangle_offset

```

Comment:
If you want to 'add' angles, you should use `local _, Ang = LocalToWorld(Vector(), hitangle_offset, Vector(), trace.HitNormal:Angle())`